[SECENG-853] Adding documentation for `settings` sub-command (#7513)
circleci/circleci-docs: jekyll/_cci2/use-the-cli-and-vcs-for-config-policy-management.adoc

The following sub-commands are currently supported within the CLI for configuration policy management:

* `diff` - shows the difference between local and remote policy bundles
* `fetch` - fetches the policy bundle (or one policy, based on name) from remote
* `push` - pushes the policy bundle (activate policy bundle)
+
For example, running the following command activates a policy bundle stored at `./policy_bundle_dir_path`:
+
[source,shell]
----
circleci policy push ./policy_bundle_dir_path --owner-id <your-organization-ID>
----

> *  `settings` - used to modify config policy management settings as required.

Reviewer: @sagar-connect I added the info about the settings subcommand up here with other sub commands, I wonder if we can add a little example for `diff` and `push` too for consistency?

Author: This will make our 'quick start' section a bit too long. Hope you agree

Follow these steps to create a policy that checks the `version` of CircleCI config files to ensure it is greater than or equal to `2.1`.

> . Enable config policy management for your organization

Reviewer: @sagar-connect I added enabling CPM for the org as the first step in the quickstart rather than a separate section. The options for this are described further down the doc too. Let me know if this seems ok

Author: Looks good to me.

circleci-cli policy push ./config-policies --owner-id <your-organization-ID>
----

[#config-policy-management-enablement]
== Enable/Disable Policy Evaluation in CI

> This option controls policy evaluation in CI build pipeline. The setting can be modified using `--enabled` flag.

Reviewer: ```suggestion
This option controls whether policy evaluation is applied to pipeline configurations within your organization. The setting can be modified using `--enabled` flag.
```

> If --enabled is set to `true`: project configuration will be evaluated against your organization's policies in CI
> If --enabled is set to `false`: project configuration will *not* be evaluated against your organization's policies in CI

Reviewer: ```suggestion
* If `--enabled` is set to `true`, project configuration **will** be evaluated against your organization's policies when pipelines are triggeres
* If `--enabled` is set to `false`, project configuration will *not* be evaluated against your organization's policies when pipelines are triggered.
```

Author: *triggered

Author: **will** is formatted with double asterisks,
*not* with single
